### ISSUE TITLE
 New Error when running report - flextable?

### ISSUE THREAD

**Issue by torv72** [torv72/torv-reports-v4#45]
Just pulled your new changes and got the following error.
```
n `mutate()`:
ℹ In argument: `text_color = case_when(...)`.
Caused by error in `case_when()`:
! Failed to evaluate the right-hand side of formula 1.
Caused by error in `color()`:
! Function `color()` supports only flextable objects.
Backtrace:
  1. ... %>% ...
 13. flextable::color(torv_color)
 14. base::stop(...)
Called from: signal_abort(cnd, .file)
Browse[1]> 

Quitting from lines 17-231 [unnamed-chunk-430] (report.Rmd)

Quitting from lines 323-337 [om-results] (report.Rmd)
```

Here is the function:
```
generate_report(.site_name = "Maroon Creek Club",
                .zip_code = 81611,
                .date_sample_submitted = "2024-09-18",
                .start_date = "2017-01-01",
                .om_seasons = "season",
                .om_stats = "median",
                .warm_or_cool = "cool",
                .acid_extract = "Mehlich",
                .include_results_interpretation = "No",
                .include_sand_fraction = "Yes",
                .draw_beeswarm = "Yes",
                .output = c("pdf", "HTML"))
```

**Comment by z3tt** [comment]
Yeah, already run into the same problem. Fixed it already, now also available for you to pull.

**Comment by torv72** [comment]
Strange for sure.  I was running reports no problem yesterday.  I pulled your changes in just a little bit ago and the error is showing up.  

Hmmm…..

**Comment by torv72** [comment]
FYI.  Tried pulling and nothing new came thru in case you did push it.  Everything is “up to date”. 

 Still running into the error.

**Comment by torv72** [comment]
Just an heads up.  Same pesky error still showing up.


❗ An executive summary for this site already exists. Sourcing the file "exec-summary_Maroon-Creek-Club_2024-09-18_season_median.txt" in the "report" directory.
    - To add comments, open the txt file, modify its content, and generate the report again.
    - To create a new auto-generated executive summary for this site, please delete the txt file, and rerun the report generation.

  |...........................................................................                               |  71% [om-results]          Error in `mutate()`:
ℹ In argument: `text_color = case_when(...)`.
Caused by error in `case_when()`:
! Failed to evaluate the right-hand side of formula 1.
Caused by error in `color()`:
! Function `color()` supports only flextable objects.
Backtrace:
  1. ... %>% ...
 13. flextable::color(torv_color)
 14. base::stop(...)
Called from: signal_abort(cnd, .file)
Browse[1]> 

Quitting from lines 17-231 [unnamed-chunk-430] (report.Rmd)

Quitting from lines 323-337 [om-results] (report.Rmd)

**Comment by z3tt** [comment]
Hm, are you still receiving the flextable error with the latest changes? I pushed the fix yesterday afternoon.

**Comment by z3tt** [comment]
Oh, that's a new one I haven't seen before. It seems it's now a reverse function matching issue (flextable and prismatic both have a function named `color()`). Can you please test if it works now?

**Comment by torv72** [comment]
When I "pull" nothing happens.  I get a message saying everything is up-to-date.  I am on the output-html branch and have always been on that.

**Comment by torv72** [comment]
I thought this would be a great time to try and figure this out myself since I couldn't get your pull to update anything.  With that being said, I figured it out!  I hope you don't mind. I only made the couple small changes to the om_testing.RMD file.   I'll push the change.  I ran it and it works!!!

I attached a snapshot from the om_testing.Rmd file.  I changed line 33 to

torv_color <- prismatic::color(torv_green)

and then I changed line 53 to

    text_color = case_when(plot_date == max(plot_date) ~ torv_color, TRUE ~ prismatic::clr_lighten(torv_color, .15))
  )
<img width="986" alt="Screenshot 2024-10-08 at 1 04 22 PM" src="https://github.com/user-attachments/assets/cf829d08-c368-4741-98e6-8191b248d2ab">

**Comment by z3tt** [comment]
Strange the changes were not visible on your end. I don't mind and your changes are fine!I also thought we can remove the `color()` function here (it was implemented by the R for the Rest of Us team) but unfortunately, it doesn't work on my end (not sure why though). So I keep my version and cross my fingers it also runs for you 🤞

We do not need the `prismatic::` namespace for `clr_lighten()` as the prismatic package is loaded anyway and there is no function name conflict (no other package with a function with the same name).

**Comment by torv72** [comment]
Yeah.  I don’t know why your pushes earlier did not work regarding this issue.  All other pushes/pulls have worked. So I set out to see if I could fix it….good practice for me.  I think we are good to go now though as everything is working and I am able to render reports.  I was able to pull this morning with your changes that you have made to date including the Sand Fraction chart!!!  I can see the light at the end of the tunnel :)
